src/Basic/NaturalTransformation.lidr

```diff
@@ -159,3 +160,72 @@ The code above is everything we need to define what a natural transformation is.
 >                             (mapObj fun2 b)
 >                             (mapMor fun1 a b f)
 >                             (component b)
+>
+> naturalTransformationExt :
+>   (cat1, cat2 : Category)
+>   -> (fun1, fun2 : CFunctor cat1 cat2)
+>   -> (trans1, trans2 : NaturalTransformation cat1 cat2 fun1 fun2)
+>   -> ((a : obj cat1) -> component trans1 a = component trans2 a)
+>   -> trans1 = trans2
+> naturalTransformationExt cat1 cat2 fun1 fun2 trans1 trans2 eq = really_believe_me()
+>
+> naturalTransformationComposition :
+>   (cat1, cat2 : Category)
+>   -> (fun1, fun2, fun3 : CFunctor cat1 cat2)
+>   -> NaturalTransformation cat1 cat2 fun1 fun2
+>   -> NaturalTransformation cat1 cat2 fun2 fun3
+>   -> NaturalTransformation cat1 cat2 fun1 fun3
+> naturalTransformationComposition cat1 cat2 fun1 fun2 fun3
+>   (MkNaturalTransformation comp1 comm1)
+>   (MkNaturalTransformation comp2 comm2) =
+>     MkNaturalTransformation
+>       (\a => compose cat2 (mapObj fun1 a) (mapObj fun2 a) (mapObj fun3 a) (comp1 a) (comp2 a))
+>       (\x, y, f =>
```

Comment:
in the live above and in other files, initial letters of the alphabet are used to denote objects. Could you use `a` and `b` here and in the definitions below?